src/core/viz/expressions/clusterAggregation.js

```diff
@@ -64,18 +76,22 @@ export const ClusterMax = genAggregationOp('max', 'number');
 export const ClusterMin = genAggregationOp('min', 'number');
 
 /**
- * Aggregate using the maximum value. This operation disables the access to the property
+ * Aggregate using the mode value. This operation disables the access to the property
```

Comment:
Maybe we should add (here and in all the other cluster functions): "except within other cluster aggregate functions" or something similar?

Reply:
:+1: